index.html

```diff
@@ -0,0 +1,121 @@
+        <meta name="viewport" content="width=device-width, initial-scale=1.0">
+        <title>Home Page</title>
+        <link rel="shortcut icon" type="image/png" href="/images/Campaign.png">
+        <script src="https://kit.fontawesome.com/7cbe454c49.js" crossorigin="anonymous"></script>
+        <link rel="stylesheet" href="./scss/index.css">
+    </head>
+    <body>
+        <header class="header-body">
+            <i class="fa fas-solid fa-bars open-menu-nav"></i>        
+                <aside class="aside-content">
+                    <nav class="whole-body-nav">
+                        <div class="top-nav"> 
+                            <nav class="close-nav">
+                                <i class="fa fa-times fa-2x time-put-up close-menu-nav"></i>
+                            </nav>
+                            <div class="top-nav-content">
+                                <i class="fa fas-brands fa-facebook-f"></i>
+                                <i class="fa fas-brands fa-twitter"></i>
+                                <a href="#">English </a>
+                                <a href="#"> My Page </a>
+                                <a href="#"> Logout </a>
+                            </div>
+                        </div>
+                    </nav>
+                    <div class="header-content">
+                        <a href="./about.html" class="center-btn-logo">
+                            <nav class="button-logo">
+                                <i class="fa fas-regular fa-globe"></i>
+                                <div class="logo-name">
+                                    <span>Olamarx's</span>
+                                    <span>Web Development</span>
+                                </div>
+                            </nav>
+                        </a>
+                        <ul class="header-middle-content">
+                            <li> <a href="/about.html">About</a> </li>
+                            <li> <a href="#"> Program</a> </li>
+                            <li> <a href="#"> Join</a> </li>
+                            <li> <a href="#">Sponsor</a> </li> 
+                            <li> <a href="#">News</a> </li>
+                        </ul>
+                        <div class="campaign-btn-center">
+                            <button type="button" class="campaign-button">
+                                <i class="fa fas-regular fa-globe"></i>
+                                <span>Campaign</span>
+                            </button>
+                        </div>
+                    </div>
+                </aside>
+        </header>
+
+        <section class="bigger-body">
+            <article class="bigger-body-bg">
+                <div class="header-text">
+                    <div class="header-text-content">
+                        <h3 class="header-text-h3">
+                            "Hello! Welcome to Olamarx's:"
+                        </h3>
```

Comment:
currently, your mobile view is like this 
project mobile :
![image](https://user-images.githubusercontent.com/60612329/163478957-e1acd9c3-ef8c-4a32-85a3-6a765d8ee290.png)
 but should have more padding for the section because it will look more visually attractive to a user 
template mobile: 
![image](https://user-images.githubusercontent.com/60612329/163479656-cb2b031b-e3c4-4b2f-ad66-2e043e3a8c1c.png)